src/main/java/Storage.java

```diff
@@ -0,0 +1,80 @@
+import java.io.*;
+import java.util.ArrayList;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    public String filePath;
+    public Ui ui;
+
+    public Storage(String filePath, Ui ui) {
+        this.filePath = filePath;
+        this.ui = ui;
+    }
+
+    /**
+     * Appends a new task to the back of the file.
+     * @param task New task specified to be added.
+     */
+    public void appendFile(Task task) {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter(filePath, true));
+            String line = task.toLine();
+            writer.write(line);
+            writer.newLine();
+            writer.close();
+        } catch (Exception ignored) {
+        }
+    }
+
+    /**
+     * Rewrites the updated list of tasks to the file.
+     * @param list Updated Tasklist of tasks.
+     */
+    public void writeFile(TaskList list) {
```

Comment:
Is this function able to write to `filePath`s that don't exist? E.g. if the user does not have the folder `data` and the `filePath` is `"data/tasks.txt"`, what should be the course of action? Perhaps you can look into the `File.mkdir()` command, which allows you to create directories, i.e. if the directory is not found, create it, but at the same time let the user know that you'll be starting from a new save file.